api/dapi.js

```diff
@@ -0,0 +1,112 @@
+'use_strict';
+const quotes = require('./quotes.js');
+const discord = require('discord.js');
+const client = new discord.Client();
+const config = require('./config.json');
+client.login(config.token)
+
+function get_color(role) {
+    switch (role) {
+        case 'OVNERS':
+            r = 'is-danger';
```

Comment:
You never define r

Reply:
Use <Collection>.get(<ID>) instead of <Collection>.find('id', <ID>);